app/uk/gov/hmrc/transitmovements/models/formats/MongoFormats.scala

```diff
@@ -53,6 +54,7 @@ trait MongoFormats extends CommonFormats with MongoBinaryFormats.Implicits with
   implicit val movementFormat: Format[Movement]                               = Json.format[Movement]
   implicit val movementWithoutMessagesFormat: Format[MovementWithoutMessages] = Json.format[MovementWithoutMessages]
   implicit val messageResponseFormat: Format[MessageResponse]                 = Json.format[MessageResponse]
+  //implicit val messageStatusFormat: Format[MessageStatus]                     = Json.format[MessageStatus]
```

Comment:
This is commented out, did it not work when you had the codec in and used val setStatus = mSet("messages.$[element].status", message.status)? If not then can we remove the commented out code?

Reply:
yes, removed this